example-client/src/main/java/com/google/cloud/dataproc/jdbc/example/DataprocClient.java

```diff
@@ -0,0 +1,121 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.dataproc.jdbc.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import java.util.Scanner;
+
+/** Example client usage of the Dataproc JDBC Driver. */
+public class DataprocClient {
+
+    public static String validateHost(String jdbcUrl, Scanner in, Properties... info)
+            throws SQLException {
+        Connection con = DriverManager.getConnection(jdbcUrl);
+        System.out.println("\nConnection established.\n");
+        Statement stmt = con.createStatement();
+
+        String tableName = "testHiveDriverTable";
+
+        stmt.execute("drop table if exists " + tableName);
+        stmt.execute(
+                "create table "
+                        + tableName
+                        + " (key int, value string, hostname string) row format delimited fields terminated by"
+                        + " ','");
+
+        // Show tables
+        String sql = "show tables '" + tableName + "'";
+        System.out.println("Running: " + sql);
+        ResultSet res = stmt.executeQuery(sql);
+        if (res.next()) {
+            System.out.println(res.getString(1));
+        }
+
+        // Describe table
+        sql = "describe " + tableName;
+        System.out.println("\nRunning: " + sql);
+        res = stmt.executeQuery(sql);
+        while (res.next()) {
+            System.out.println(res.getString(1) + "\t" + res.getString(2));
+        }
+
+        // Load data into table
+        // NOTE: filepath has to be local to the hive server
+        // NOTE: /tmp/input.txt is a comma separated file with two fields per line
+        String filepath = "/tmp/input.txt";
+        sql = "load data local inpath '" + filepath + "' into table " + tableName;
+        System.out.println("\nRunning: " + sql);
+        stmt.execute(sql);
+
+        // select * query
+        sql = "select hostname from " + tableName + " where key = 1";
+        System.out.println("\nRunning: " + sql);
+        res = stmt.executeQuery(sql);
+        String host = null;
+        if (res.next()) {
+            host = res.getString(1);
+            System.out.println("Host name is: " + host);
+        }
+
+        System.out.println("\nDo you want to execute another query? [y/n]");
+        String response = in.nextLine();
```

Comment:
Nit: if response ends with a semicolon, let's strip it out and then run the query (just as a usability thing)